packages/cli-kit/src/private/node/analytics.ts

```diff
@@ -97,9 +97,20 @@ export async function getEnvironmentData(config: Interfaces.Config): Promise<Env
 export async function getSensitiveEnvironmentData(config: Interfaces.Config) {
   return {
     env_plugin_installed_all: JSON.stringify(getPluginNames(config)),
+    env_shopify_variables: JSON.stringify(Object.fromEntries(getShopifyEnvironmentVariables())),
   }
 }
 
+function getShopifyEnvironmentVariables() {
+  const shopifyEnvVars = new Map<string, string>()
+  for (const [key, value] of Object.entries(process.env)) {
+    if (key.startsWith('SHOPIFY_') && value !== undefined) {
+      shopifyEnvVars.set(key, value)
+    }
+  }
+  return shopifyEnvVars
+}
```

Comment:
Just make `getShopifyEnvironmentVariables` return a JSON, this function can be simplified to a single line:
```ts
Object.fromEntries(Object.entries(process.env).filter(([key]) => key.startsWith('SHOPIFY_')))
```